packages/dashboard/src/components/alert-store.tsx

```diff
@@ -1,77 +1,361 @@
-import { ApiServerModelsTortoiseModelsAlertsAlertLeaf as Alert } from 'api-client';
+import {
+  AlertRequest,
+  ApiServerModelsAlertsAlertRequestTier,
+  LogEntry,
+  TaskEventLog,
+  ApiServerModelsRmfApiLogEntryTier as LogEntryTier,
+} from 'api-client';
 import { AppEvents } from './app-events';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+  TextField,
+  Theme,
+  Divider,
+  useMediaQuery,
+  DialogContent,
+} from '@mui/material';
+import { makeStyles, createStyles } from '@mui/styles';
 import React from 'react';
+import { base } from 'react-components';
+import { AppControllerContext } from './app-contexts';
 import { RmfAppContext } from './rmf-app';
 import { Subscription } from 'rxjs';
-import { TaskAlertDialog } from './tasks/task-alert';
-
-// This needs to match the enums provided for the Alert model, as it is not
-// provided via the api-client since tortoise's pydantic_model_creator is used.
-enum AlertCategory {
-  Default = 'default',
-  Task = 'task',
-  Fleet = 'fleet',
-  Robot = 'robot',
+import { TaskCancelButton } from './tasks/task-cancellation';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    textField: {
+      background: theme.palette.background.default,
+      '&:hover': {
+        backgroundColor: theme.palette.background.default,
+      },
+    },
+  }),
+);
+
+interface AlertDialogProps {
```

Comment:
missing export

Reply:
it is only being used within `alert-store`, does it need an export?

Reply:
I missed that `AlertDialog` is private, no need to export then.